packages/pdf-viewer/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@joplin/pdf-viewer",
+  "version": "1.0.0",
+  "description": "Provides embedded PDF viewers for Joplin",
+  "main": "dist/main.js",
+  "types": "src/main.ts",
+  "publishConfig": {
+    "access": "public"
```

Comment:
I don't think it is public since we don't want to publish this package separately.